airgun/entities/computeprofile.py

```diff
@@ -67,3 +83,16 @@ def step(self, *args, **kwargs):
         self.parent.table.row(
             name=entity_name)['Actions'].widget.fill('Rename')
         self.parent.actions.fill('Rename')
+
+
+@navigator.register(ComputeProfileEntity, 'Edit')
+class EditComputeProfile(NavigateStep):
+    VIEW = ResourceProviderEditView
+
+    def prerequisite(self, *args, **kwargs):
+        return self.navigate_to(ComputeResourceEntity, 'All', **kwargs)
```

Comment:
so for `EditComputeProfile` you want to go to `ComputeResourceEntity` ?
That is also invalid